packages/flutter/lib/src/widgets/inherited_model.dart

```diff
@@ -160,6 +160,10 @@ abstract class InheritedModel<T> extends InheritedWidget {
     // Create a dependency on all of the type T ancestor models up until
     // a model is found for which isSupportedAspect(aspect) is true.
     final List<InheritedElement> models = _findModels<T>(context, aspect).toList();
+    if (models.isEmpty) {
+      return null;
+    }
```

Comment:
please add a test for this, and update the documentation accordingly. (maybe this belongs in a separate pr?)

Reply:
I actually ended up not using it. But we should fix this anyway (for consistency with InheritedWidget). Added a test.